spec/index.bs

```diff
@@ -1533,7 +1533,7 @@ interface GPURenderPassEncoder {
     void setBlendColor(GPUColor color);
     void setStencilReference(unsigned long reference);
 
-    void executeBundles(sequence<GPURenderBundle> bundles);
+    void executeBundle(GPURenderBundle bundle);
```

Comment:
Not sure how much value this gives: `GPURenderBundle` would contain a lot of WebGPU commands so the cost of converting a list from JS to C++ would be amortized.

Reply:
I'm actually not sure how well it would amortize - accessing a single element of the array may or may not be cheaper than a whole executeBundle call. I'd guess it's a tossup.